### PR TITLE
Adding more inputs for the stable video diffusion model

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

**Review by squidarth — APPROVED** [review]
thanks!

File: stable-diffusion/stable-video-diffusion/model/model.py [hunk 1]
```diff
@@ -48,16 +48,25 @@ def load(self):
 
     def predict(self, model_input: Dict) -> Dict:
         image_b64 = model_input.get("image")
-        duration = int(model_input.get("duration"))
-        if duration > 10:
-            return {"output": "The duration of the clip cannot be greater than 10"}
+        num_frames = int(model_input.get("num_frames", 14))
```
[the remark this answers is not in the document]
Comment (htrivedi99) [reply stable-diffusion/stable-video-diffusion/model/model.py:51]: Those variables aren't constant. You can change them for each request. Where should I move them to?

File: stable-diffusion/stable-video-diffusion/README.md [hunk 2]
```diff
@@ -58,7 +61,12 @@ def image_to_base64(file_path):
 
     return base64_string
 
-data = {"image": image_to_base64("path/to/image/cheetah.jpeg"), "duration": 5}
+data = {
+    "image": image_to_base64("path/to/image/cheetah.jpeg"),
+    "num_frames": 14,
```
[the remark this answers is not in the document]
Comment (htrivedi99) [reply stable-diffusion/stable-video-diffusion/README.md:66]: You would include them I believe. The variables `num_frames`, `fps`, `num_steps`, etc, allow you to control the output. They don't have to be fixed constants.

[the remark this answers is not in the document]
Comment (squidarth) [reply stable-diffusion/stable-video-diffusion/README.md:66]: right

File: stable-diffusion/stable-video-diffusion/model/model.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (squidarth) [reply stable-diffusion/stable-video-diffusion/model/model.py:51]: I mean the default values like "14" and not. What I envisioned was a DEFAULT_NUM_FRAMES, etc. comments. tioli though, I think this is fine

[the remark this answers is not in the document]
Comment (htrivedi99) [reply stable-diffusion/stable-video-diffusion/model/model.py:51]: Oh, I see what you mean. Yeah, I can make them constants

**Pull request merged**